src/matrMineral.ttl

```diff
@@ -83,10 +83,20 @@ somamin:NonmetallicMineral rdf:type owl:Class ;
                        rdfs:label "nonmetallic mineral"@en .
 
 
+###  http://sweetontology.net/matrMineral/Olivine
+somamin:Olivine rdf:type owl:Class ;
+           rdfs:subClassOf somamin:Mineral ;
+           rdfs:label "olivine"@en ;
+           dcterms:contributor <https://orcid.org/0000-0002-0337-8610> .
+
+
 ###  http://sweetontology.net/matrMineral/Olvine
 somamin:Olvine rdf:type owl:Class ;
            rdfs:subClassOf somamin:Mineral ;
-           rdfs:label "olvine"@en .
+           rdfs:label "olvine"@en ;
+           owl:deprecated "true"^^xsd:boolean ;
+           rdfs:seeAlso somamin:Olivine ;
+           dcterms:contributor <https://orcid.org/0000-0002-0337-8610> .
```

Comment:
Add 
```
dcterms:isReplacedBy  <http://sweetontology.net/matrMineral/Olivine> ;
```

Reply:
added; also removed `rdfs:seeAlso somamin:Olivine ;`

Reply:
added  dcterms:isReplacedBy  somamin:Olivine ;   (instead of full path)